tethys_cli/scaffold_templates/app_templates/react/reactapp/services/utilities.js

```diff
@@ -1,13 +1,30 @@
-function getTethysPortalHost() {
-    let tethys_portal_host = process.env.TETHYS_PORTAL_HOST;
-
-    // If the .env property is not set, derive from current location
-    if (!tethys_portal_host || !tethys_portal_host.length) {
-        let currLocation = window.location.href;
-        let url = new URL(currLocation);
-        tethys_portal_host = url.origin;
-    }
-    return tethys_portal_host;
+const APP_ROOT_URL = process.env.TETHYS_APP_ROOT_URL;
+
+export function getTethysPortalHost() {
+  let tethys_portal_host = process.env.TETHYS_PORTAL_HOST;
+  let tethys_prefix_url = process.env.TETHYS_PREFIX_URL.replace(/^\/|\/$/g, "");
+
+  // If the .env property is not set, derive from current location
+  if (!tethys_portal_host || !tethys_portal_host.length) {
+    let currLocation = window.location.href;
+    let url = new URL(currLocation);
+    tethys_portal_host = url.origin;
+  }
+
+  if (tethys_prefix_url) {
+    return `${tethys_portal_host}/${tethys_prefix_url}`;
```

Comment:
Why are you adding prefix url to the host? The host should be the origin only. If this is needed, create another function that combines the TethysPortalHost and Prefix: `getTethysPortalHostWithPrefix()`.